app.js

```diff
@@ -0,0 +1,92 @@
+const billAmount = document.querySelector(".bill-amount");
+const tips = document.querySelectorAll(".tip");
+const customTip = document.querySelector(".custom");
+const numberOfPeople = document.querySelector(".people-tip");
+const tipAmount = document.querySelector(".total-tip");
+const totalTip = document.querySelector(".total");
+const reset = document.querySelector("button");
```

Comment:
I think this variable could be renamed to resetButton to be more suggestive that it refers to the button in the ui.

Reply:
Done!